### ISSUE TITLE
Bug Fix: Favorites Container with 1 and 2 items

### ISSUE THREAD

**Issue by ivonne-hernandez** [isleofyou/lemon-aide#12]
As a user, when I only have 1 or 2 favorited products I should see those products displayed on the favorites page correctly. 

- [x] Update display of 1 favorited product:

<img width="1437" alt="Screen Shot 2022-01-09 at 7 04 11 PM" src="https://user-images.githubusercontent.com/86989161/148710944-004a4a37-9e8e-49cb-a96f-2e10d086c4bc.png">

- [x] Update display of 2 favorited products:

<img width="1440" alt="Screen Shot 2022-01-09 at 7 03 47 PM" src="https://user-images.githubusercontent.com/86989161/148710952-89b5816c-0da5-4477-83ca-dd49e7c1a0f2.png">

**Comment by ivonne-hernandez** [comment]
This is what it will look like once the Refactor/favorites branch is merged into main:

Updated display of 1 favorited product:
<img width="1440" alt="Screen Shot 2022-01-10 at 10 35 52 AM" src="https://user-images.githubusercontent.com/86989161/148812438-99f2cff0-71bf-403e-98c0-df7252a0b336.png">

Updated display of 2 favorited products:
<img width="1440" alt="Screen Shot 2022-01-10 at 10 36 15 AM" src="https://user-images.githubusercontent.com/86989161/148812463-3d7d46c4-613d-464e-b51b-f80ea297b79c.png">

**Comment by ivonne-hernandez** [comment]
David, we may have this same issue given the grid refactoring on your footer branch. Please let me know if that is the case and if so, if you'd be able to work on a fix for it?

**Comment by isleofyou** [comment]
Bingo
